core/src/main/java/io/jenkins/pluginhealth/scoring/probes/SCMLinkValidationProbe.java

```diff
@@ -118,8 +121,8 @@ public String[] getProbeResultRequirement() {
      * @return folderPath if it valid or return the scm itself
      */
     private String searchPomFiles(Path directory, String pluginName, String scm) {
-        try (Stream<Path> paths = Files.find(directory, 2, (path, $) -> path.getFileName().toString().equals("pom.xml"))) {
-            return paths.filter(pom -> pomFileMatchesPlugin(pom, pluginName)).toString();
+        try (Stream<Path> paths = Files.find(directory.resolve("pom.xml"), 2, (path, $) -> path.getFileName().toString().equals("pom.xml"))) {
+            return paths.filter(pom -> pomFileMatchesPlugin(directory, pluginName)).toString();
```

Comment:
How do I collect the Paths? It is is of type `Path` and not String. So the `Collectors.toList` and `Collectors.joining` do not work.